client/src/PageSizeMenu.tsx

```diff
@@ -18,7 +18,12 @@ const PageSizeMenu: AppFunctionComponent<InputProps, typeof mapStateToProps> = (
   return (
     <span>
       Results per page:
-      <DropdownButton variant="info" title={`${pageSize}`} id="page-size-menu">
+      <DropdownButton
+        as={ButtonGroup}
+        variant="info"
+        title={`${pageSize}`}
```

Comment:
maybe i shouldn't ask but is this a stupid react thing? you can't just write `title={pageSize}`？

Reply:
Looks like it doesn't auto-cast to a string in that case.